### DISCUSSION TITLE
Bug - glycemia history

### DISCUSSION THREAD

**Discussion by AliQ101** [NightscoutFoundation/xDrip#2410] [General]
It seems that there is something wrong in glycemia history view since Nightly build 13th Oct 2022 (2022.10.13), implementing companion bluetooth feature (no matter if it is on/off). I can see full detail history only for 30 days back (until 13th Sep) and then each day back is almost double short, no possibility to display before July 2022. By using the latest build (2022.10.31.) I can still properly view glycemia history also only until 13th Sep. Is it bug or I missed something?

![Screenshot_20221101-154524](https://user-images.githubusercontent.com/107270312/199263385-35871495-137e-42bd-a43a-06750f39f202.png)
![Screenshot_20221101-154537](https://user-images.githubusercontent.com/107270312/199263375-a6895a12-b55d-43d8-9132-78143f0332f0.png)
![Screenshot_20221101-154616](https://user-images.githubusercontent.com/107270312/199263382-9cc42ea7-35a3-4472-a3cf-bd74102b16c7.png)

**Comment by Navid200** [comment]
I see a similar issue.

What do you mean by "since October 13, 2022"?
Are you saying if you install the October 5, 2022 version, you don't see this behavior?

**Reply by AliQ101** [reply]
Yes, I tried Nightly 5th October and its ok, the issue started from Nightly 13th October.

**Reply by Navid200** [reply]
Just a sec please

**Reply by Navid200** [reply]
@AliQ101 
I have a build of August 7, 2022 that does not have the problem.
And all the builds I tried after August 21, 2022 show the problem.

So, I am puzzled why October 5, 2022 works for you.

Would you please make sure about that because it is very important to show which version first had the problem.
After you are certain you have the answer, please go ahead and open an issue.
You can copy and paste your first post from this thread as the post for the issue if you like to save you time.
Or, right it from scratch.

But, please make sure two items are included in the first post of the issue:
1- A link to this discussion thread
2- A clarification of which two consecutive official versions show the start of this issue, where the first does not have the problem while the next does.

**Reply by Navid200** [reply]
And thanks for reporting

**Reply by AliQ101** [reply]
@Navid200 I can not see any build of August 7, 2022 as you mentioned above.
I went through the Nightly releases chain and Nightly build 3rd Aug 2022, stable, e13a76e-2022.08.03 is the last version that is ok. Next Nightly build 18th Aug 2022, c649f9d-2022.08.18., has started the issue in glycemia history view.
I will open the issue.

**Reply by Navid200** [reply]
Sorry about that.  I have personal builds.  But, the dates now make sense.
So, if you have identified two consecutive official builds where the first one does not have the problem, and the second one has the problem, if you don't mind, please open an issue as I described.

Thanks so much for spending time on this.
